### PR TITLE
feat: documentação rota /api/pets/breeds/ GET # issue-177

### LINKED ISSUE #177: Documentar as rotas sob o domínio/contexto de /api/pets/breeds/ GET
**Describe the feature**
Documentar as rotas sob o domínio/contexto de /api/pets/breeds/ GET

<details open>  
  <summary>  
    <b>Description</b>  
  </summary>  
  Documentar o endpoint que retorna uma lista de raças de pets. Incluir detalhes sobre os parâmetros de consulta (se houver), formato de resposta, exemplos de uso, e status codes possíveis.  
</details>  
<br>  
<details>  
  <summary>  
    <b>Acceptance Criteria</b>  
  </summary>  
A documentação deve conter uma descrição clara da rota.<br/>  
Exemplo de requisição e resposta.<br/>  
Explicação dos status codes.<br/>  
</details>  
<br>  
<details>  
  <summary>  
    <b>Implementation Details</b>  
  </summary>  
Está no MakeFiles  
swag:  
	swag init -g api/main.go -o swagger/  
</details>  
<br>  
<details>  
  <summary>  
    <b>Visual Concepts</b>  
  </summary>  
[Include any visual representations or concepts if those are available and applicable.]  
</details>  
<br>  
<details>  
  <summary>  
    <b>Additional Information</b>  
  </summary>  
[Provide any additional information, such as context that might be relevant to the implementation of this specific feature.]  
</details>  
<br>  
<details>  
  <summary>  
    <b>Impedimentos</b>  
  </summary>  
Ter ferramenta de documentação implementada  
Issue: https://github.com/devhatt/pet-dex-backend/issues/119  
</details>  
<br>

### PR DESCRIPTION
**Pull request by EdiltonOliveira**
Closes #177 

<details open> 
  <summary>
    <b>Feature</b>
  </summary>

* Adiciona documentação para a rota api/breeds/
</details>

<details open> 
  <summary>
    <b>Bugfix</b>
  </summary>

- **Description**
N/A

- **Cause**
N/A

- **Solution**
N/A
</details>

<details> 
  <summary>
    <b>Changelog</b>
  </summary>
N/A
</details>

<details open> 
  <summary>
    <b>Visual evidences :framed_picture:</b>
  </summary>

</details>

Captura de tela de 2024-09-09 14-24-06

<details open> 
  <summary>
    <b>Checklist</b>
  </summary>

  - [x] Issue linked
  - [ ] Build working correctly
  - [x] Tests created
</details>

<details> 
  <summary>
    <b>Additional info</b>
  </summary>
N/A
</details>

### CODE REVIEW

**Comment by Luiginicoletti** [comment]
@PiluVitu 

Screenshot 2024-09-09 at 15 16 46

**Comment by PiluVitu** [comment]
> @PiluVitu 
> 
> Screenshot 2024-09-09 at 15 16 46
> 

Quando acontecer isso pae, é so assignar o dono no pr e rodar a CI dnv, essa semana to ajustando isso

File: api/controllers/breed.go
[the remark this answers is not in the document]
Comment (EdiltonOliveira) [reply api/controllers/breed.go:1]: Obrigado pelas correções.

**Review by Luiginicoletti — APPROVED** [review]
Não entendo muito mas parece bem estruturado!

**Review by davidambz — APPROVED** [review]

**Review by juliaam — APPROVED** [review]

**Pull request merged**